nats-server/src/lib.rs

```diff
@@ -24,6 +24,13 @@ use rand::Rng;
 use regex::Regex;
 
 pub struct Server {
+    inner: Inner,
```

Comment:
Why the indirection?

Reply:
Because if you would assing new server directly to *self, drop() is called, which wipes the data etc. i want to keep wiping data on drop.
Other solution is to return big tuple from run_server internal fn, but thats ugly.